.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,20 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [master]
```

Comment:
Do you mind naming the main branch `main` instead of `master`? (common practice nowadays to avoid the master-slave connotation of "master")

Reply:
It's already the case, this is actually a typo indeed

Reply:
That makes sense. I was surprised actually :sweat_smile: